<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="ipAddressLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IP Address of Server:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipAddress">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>30</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="portLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>35</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>port:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>78</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>78</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="gameNameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Game</string>
   </property>
  </widget>
  <widget class="QLabel" name="userlabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>faction IDl</string>
   </property>
  </widget>
  <widget class="QLabel" name="passLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gameName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="userId">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="port">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
